iroha/src/torii.rs

```diff
@@ -167,9 +167,6 @@ impl Torii {
         server.at(uri::QUERY_URI).get(handle_queries);
         server.at(uri::HEALTH_URI).get(handle_health);
         server.at(uri::METRICS_URI).get(handle_metrics);
-        server
-            .at(uri::PENDING_TRANSACTIONS_ON_LEADER_URI)
-            .get(handle_pending_transactions_on_leader);
```

Comment:
Was there a duplicate?

Reply:
I guess we need to create an issue to disallow duplicate handlers. Can you do it?